.github/workflows/md-external-links-check.yml

```
@@ -0,0 +1,40 @@
+name: Check external links
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "20 4 * * 1"
+permissions:
+  contents: read
+  issues: write
+jobs:
+  markdown-link-check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: gaurav-nelson/github-action-markdown-link-check@v1
+        with:
+          use-quiet-mode: 'yes'
+          base-branch: 'main'
+          config-file: '.github/markdown-external-links.config.json'
+  open-issue:
+    runs-on: ubuntu-latest
+    needs: [markdown-link-check]
+    if: failure()
+    steps:
+      - uses: actions/checkout@v4
+      - name: Compute Run URL
+        uses: actions/github-script@v7
+        id: compute-run-url
+        with:
+          script: return `https://github.com/${context.repo.owner}/${context.repo.repo}/actions/runs/${context.runId}/attempts/${context.runAttempt || 1}`;
+          result-encoding: string
+
+      - name: Create an issue
+        uses: JasonEtco/create-an-issue@v2
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          RUN_URL: ${{ steps.compute-run-url.outputs.result }}
```

```suggestion
          RUN_URL: ${{ format('https://github.com/{0}/{1}/actions/runs/{3}/attempts/{4}', context.repo.owner, context.repo.repo, context.runId, context.runAttempt || 1) }}
```
Would not do the job?

---

unfortunately, no.
To access `context` we need to use `actions/github-script`

---

but you do have access to `GITHUB_REPOSITORY_OWNER`, `GITHUB_RUN_ATTEMPT` and `GITHUB_RUN_ID` `GITHUB_REPOSITORY` .
If I remember well, `context` variable is actually populated with the `github` envvar